### PR TITLE
React setup

### PR DESCRIPTION
**Pull request by dmoygit**
In this PR:
Client folder
1. index.jsx:  added sample data from experience 4 in my db
2. app.jsx: passing all data to the appropriate components
3. pageSelector.jsx: received the current page number (each page will only show 5 reviews)
4. reviewList.jsx: passes thru 1 review at a time to the ReviewListItem component and groups the whole reviews section to float to the right of the page.
5. reviewListItem.jsx:  renders the avatar, username, date, number of stars, and the review.
Notes: I only put in minor css to make sure each section stayed in place. I still need to get correct formatting for the data, rendering stars, etc. the purpose of this PR is to just setup the React "plumbing"

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReviewList from './reviewList.jsx';
+import PageSelector from './pageSelector.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      allPreviews: props.reviews,
```
Comment (yerx) [inline client/components/app.jsx:9]: It looks like your code works. I thought that for class components you would have to use "this.props.reviews" and for functional components use "props.reviews".

File: client/components/app.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      overallStar: 3.75,
+      currPage: 1
+    }
+
+  }
+
+  render() {
+    let reviews = this.state.allPreviews.slice((this.state.currPage - 1) * 5, this.state.currPage * 5);
```
Comment (yerx) [inline client/components/app.jsx:17]: Do the reviews match up with the specific experience_id that the user is viewing? 

OK, I see further down that you are pulling reviews only from experience_id = 4.

**Review by yerx — APPROVED** [review]
Lgtm. I had a comment about pulling reviews based on experience_id, but saw that you are already taking that into account. Approved. Let me know if you have any questions.

File: client/components/app.jsx (hunk 1 above)
Comment (dmoygit) [reply client/components/app.jsx:9]: I believe that's only when you're passing props into the class's functions

**Pull request merged**